KsApi/models/graphql/adapters/Comment+GraphComment.swift

```diff
@@ -4,26 +4,27 @@ extension Comment {
   /**
    Returns a minimal `Comment` from a `GraphComment`
    */
-
-  /**
-   FIXME: Some of the properties needed for `Comment` hasn't been mapped out on GraphComment. We are returning default values.
-   These properties are `Author.imageUrl`, `authorBadges`, `createdAt`, `deletedAt`, `isDeleted`
-   */
   static func comment(from graphComment: GraphComment) -> Comment {
+    var authorBadges = [Comment.Author.AuthorBadge]()
```

Comment:
Likewise with the deserialization here, just need to conform `Badge` to `String, Decodable`.

Reply:
I changed over `Badge` to be `String, Decodable` but I still need to conver the `GraphComment.GraphBadge` to an `Comment.AuthorBadge` because the deserializer works for JSON only as I understand it. I simplified the conversion though, will commit the change shortly.